services/checks/checks.go

```diff
@@ -244,6 +246,31 @@ func (s *Service) waitForResult(ctx context.Context, resultID string) ([]map[str
 	}
 }
 
+// getMinAgentVersionForCheckType returns minimum version that pmm-agent
+// should has to be able process action of passed type.
+func getMinAgentVersionForCheckType(t check.Type) (*version.Parsed, error) {
+	switch t {
+	case check.MySQLSelect:
+		fallthrough
+	case check.MySQLShow:
+		fallthrough
+	case check.PostgreSQLSelect:
+		fallthrough
+	case check.PostgreSQLShow:
+		fallthrough
+	case check.MongoDBBuildInfo:
+		fallthrough
+	case check.MongoDBGetParameter:
+		return version.Parse("2.6.0")
+
+	case check.MongoDBGetCmdLineOpts:
+		return version.Parse("2.6.0")
```

Comment:
That should be 2.7.0.

Those also could be global variables so we don't have to re-parse them every time.

Reply:
Damn, left after testing 🤦‍♂️